src/task/detect-bad-words-task.ts

```diff
@@ -27,6 +27,10 @@ export class DetectBadWordsTask extends BaseValidationTask<string[]> {
   checkBadWrods = (documents: contentForValidation[]) => {
     const contents = documents?.filter(Boolean);
     const badWordsFilter = new BadWordsFilter();
+    // this package does not have a TS one, so I have to use 'require' here
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const frenchBadwordsList = require('french-badwords-list').array as Array<string>;
+    badWordsFilter.addWords(...frenchBadwordsList);
```

Comment:
Perhaps we want to factor out the  building of the list -> so it's easier to find if we want to add words. Could you create a new file for that?